string name
phoxi_camera/DeviceType type
string hwIdentification
string IPaddress
phoxi_camera/DeviceConnectionStatus status
string firmwareVersion
string variant
bool isAlpha
bool isColor